### ISSUE TITLE
Don't force system virtual_ips change.

### ISSUE THREAD

**Issue by DisasteR** [vesposito/easyucs#15]
We do base ip setup before using easyucs.
And we want to avoid any risk of touching IP config.

```--- a/schema/ucs/ucsm/config/system.json
+++ b/schema/ucs/ucsm/config/system.json
@@ -44,20 +44,5 @@
          "default":""
       }
    },
-   "required":[
-      "name"
-   ],
-   "anyOf":[
-      {
-         "required":[
-            "virtual_ip"
-         ]
-      },
-      {
-         "required":[
-            "virtual_ipv6"
-         ]
-      }
-   ],
    "additionalProperties":false
 }
```

**Comment by vesposito** [comment]
Hey Benjamin,
This is something we have discussed, and is already included for the next release of EasyUCS.
Unfortunately I am waiting for the next update of UCS SDKs to be officially included in their respective repos (PRs have already been published but not merged yet - see https://github.com/CiscoUcs/ucsmsdk/pull/202) before I can move to the next release, as it has dependencies with those new SDKs.

If required I can push that specific patch in the meantime.

**Comment by DisasteR** [comment]
ok that's a good news :)
i have created the issue to not forget.

**Comment by vesposito** [comment]
Fixed in [0.9.7.4](https://github.com/vesposito/easyucs/commit/70c66d4565fe1726f460482e813b6351ca2ccc7e)

I finally decided to push a new release without waiting for the new UCS SDKs. Mostly bug fixes and small features.
Once the UCS SDKs are updated, I will push a new release of EasyUCS with support for 4.1(2)+ features.
